src/elements/content-sidebar/activity-feed/annotations/AnnotationActivity.js

```diff
@@ -71,10 +73,28 @@ const AnnotationActivity = ({
         onEdit(id, text, permissions);
     };
 
+    const handleMenuClose = (): void => {
+        setIsMenuOpen(false);
```

Comment:
Given that these event handlers are pretty numerous, could we either inline them in the JSX or shorten the helpers?

```
const handleMenuClose = (): void => setIsMenuOpen(false);
const handleMenuOpen = (): void => setIsMenuOpen(true);